microsoft-edge/webview2/concepts/distribution.md

```diff
@@ -123,7 +121,13 @@ Only one installation of the Evergreen WebView2 Runtime is needed for all Evergr
 
 *   For offline clients: _WebView2 Runtime Standalone Installer_ is a full installer that installs the Evergreen WebView2 Runtime in offline environments.
 
-Currently, both the bootstrapper and standalone installer only support per-machine installs, which requires elevation of permissions.  If an installer is run without elevation, the user is prompted to elevate permissions.
+### Installing the Runtime as per-machine or per-user
+
+<!-- keep the 3 instances or variants of this passage in sync: -->
+
+The latest bootstrapper and standalone installer support both _per-machine_ and _per-user_ installs of the WebView2 Runtime.
+
+If you run the installer from an elevated process or command prompt, the Runtime is installed as _per-machine_.  If you don't run the installer from an elevated process or command prompt, the Runtime will be installed as _per-user_.  However, a _per-user_ install is automatically replaced by a _per-machine_ install, if a _per-machine_<!--if updater is always per-machine, strike that "per-machine" qualifier, and break out a separate statement or phrase, that the updater is per-machine--> Microsoft Edge Updater is in place.  A _per-machine_<!--strike "A _per-machine_" if updater is always per-machine--> Microsoft Edge Updater is included as part of Microsoft Edge, except for the Canary preview channel of Microsoft Edge.
```

Comment:
@liminzhu - In the 3 versions of this paragraph in this article, please remove the HTML comments `<!-- -->` I added.  Please clarify whether the updater is inherently per-machine, or if there are two versions of the updater: a per-machine updater and a per-user updater.